VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -416,6 +416,20 @@ public void setTitle(CharSequence title) {
         }
     }
 
+    public void setActionBar(boolean visible){
```

Comment:
Maybe this should be called ``setActionBarVisible``. This name makes me think it sets the action bar itself.